### PR TITLE
feat(BUX-697): remove admin dashboard

### PR DESCRIPTION
**Pull request by pawellewandowski98**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

# Description

- Removed Admin Dashboard

image

### CODE REVIEW

**Review by kuba-4chain — APPROVED** [review]

File: src/routes.js [hunk 1]
```diff
@@ -35,15 +34,9 @@ export const AppRouter = () => {
...
         <Route exact path="/transaction-new" name="New Transactions" element={<TransactionNew/>}/>
         <Route exact path="/access-keys" name="Access Keys" element={<AccessKeys/>}/>
-        {(!!xPub || !!accessKeyString)
-          ?
-          <Route exact path="/" name="xPub" element={<XPub/>}/>
-          :
-          <>
-            <Route exact path="/" name="Admin dashboard" element={<Dashboard/>}/>
-          </>
+        {(!!xPub || !!accessKeyString) &&
```
Comment (arkadiuszos4chain) [inline src/routes.js:37]: What is the 'index' view if user sign in with xPriv?

File: src/pages/dashboard.js [hunk 2]
```diff
@@ -1,153 +0,0 @@
...
-import BitcoinIcon from '@mui/icons-material/CurrencyBitcoin';
-import LocationSearchingIcon from '@mui/icons-material/LocationSearching';
-import PaymailIcon from '@mui/icons-material/Message';
-
-import { AdminChart } from '../components/dashboard/chart';
-import { DashboardLayout } from '../components/dashboard-layout';
-import { useUser } from "../hooks/user";
-import AdminLogin from "./admin-login";
-import { AdminCard } from "../components/dashboard/card";
-import { UtxosByType } from "../components/dashboard/utxos-by-type";
```
Comment (dorzepowski) [inline src/pages/dashboard.js:14]: I believe that those components should also be removed

File: src/pages/dashboard.js [hunk 3]
```diff
@@ -1,153 +0,0 @@
-import React, { useEffect, useState } from 'react';
-import {Alert, Box, Container, Grid} from '@mui/material';
-
-import ViewListIcon from "@mui/icons-material/ViewList";
-import BitcoinIcon from '@mui/icons-material/CurrencyBitcoin';
-import LocationSearchingIcon from '@mui/icons-material/LocationSearching';
-import PaymailIcon from '@mui/icons-material/Message';
-
-import { AdminChart } from '../components/dashboard/chart';
```
Comment (dorzepowski) [inline src/pages/dashboard.js:9]: I assume that we have some dependencies in package.json that were added only to display a chart,
please ensure that they are removed if not used after deletion of this component.

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: src/routes.js (hunk 1 above)
Comment (pawellewandowski98) [reply src/routes.js]: First from the navigation so adding new xpub `/admin/register-xpub`

File: src/pages/dashboard.js (hunk 2 above)
Comment (pawellewandowski98) [reply src/pages/dashboard.js:14]: Removed

File: src/pages/dashboard.js (hunk 3 above)
Comment (pawellewandowski98) [reply src/pages/dashboard.js:9]: Removed chart-js dependency

**Review by arkadiuszos4chain — APPROVED** [review]

**Review by dorzepowski — APPROVED** [review]

**Pull request merged**